composer.json

```diff
@@ -3,19 +3,37 @@
     "type": "library",
     "require": {
         "php": "^8.0",
+        "ext-json": "*",
```

Comment:
This requirement is not further required as we are on php ^8.0

[PHP 8 ext-json](https://php.watch/versions/8.0/ext-json)